camerasdk/src/main/java/com/qompium/fibricheck/camerasdk/FibriCheckerImpl2.java

```diff
@@ -224,7 +225,9 @@ public void onCaptureCompleted (CameraCaptureSession session, CaptureRequest req
 
   @TargetApi (21) private static Size chooseVideoSize (Size[] choices) {
 
-    return choices[choices.length - 1];
+    Arrays.sort(choices, (s1, s2) -> Long.compare(s1.getWidth() * (long) s1.getHeight(), s2.getWidth() * (long) s2.getHeight()));
```

Comment:
width is long and height not?

Reply:
it's a casting, if one of the two is casted, it is good for the result!